main-settings/src/main/scala/sbt/dsl/LinterLevel.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt.dsl
+
+sealed trait LinterLevel
+object LinterLevel extends LinterLevelLowPriority {
+  implicit case object Warn extends LinterLevel
```

Comment:
There should be an explanation of why this is made implicit, and how this is meant to be used.

Reply:
I force pushed an update that added scaladoc to these.